.github/workflows/deploy-beta.yml

```diff
@@ -0,0 +1,78 @@
+name: deploy beta
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: Install SSH key
+      uses: shimataro/ssh-key-action@v2
+      with:
+        key: ${{ secrets.SSH_KEY }}
+        name: dim.rsa # optional
+        known_hosts: ${{ secrets.REMOTE_HOST }}
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2 # So packtracker can get the previous commit
+
+    - name: Cache node_modules
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-node_modules
+      with:
+        path: $GITHUB_WORKSPACE/node_modules
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock')}}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock')}}
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+
+    - name: Cache eslint
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-eslint
+      with:
+        path: ~/.cache
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/eslint.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/eslint.json') }}
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+
+    - name: Install
+      uses: bahmutov/npm-install@v1
+
+    - name: Test
+      run: yarn test
+
+    - name: Lint check
+      run: |
+        find . -type d -name "node_modules" -prune -o -type f -iregex '.*.ts\|.*.js\|.*.tsx\|.*.jsx' -exec ./build/set-mtime-to-md5.sh {} \;
+        yarn run lint-check
+
+    - name: Inject slug/short variables
+      uses: rlespinasse/github-slug-action@v3.x
+
+    - name: Set BRANCH_NAME ENV VAR
+      run: echo "BRANCH_NAME=${GITHUB_REF_SLUG} >> $GITHUB_ENV"
+
+    - name: Build and deploy
+      run: ./build/ga_build.sh
+      env:
+        PT_PROJECT_TOKEN: ${{ secrets.PT_PROJECT_TOKEN }}
+        WEB_API_KEY: ${{ secrets.WEB_API_KEY }}
+        WEB_OAUTH_CLIENT_ID: ${{ secrets.WEB_OAUTH_CLIENT_ID }}
+        WEB_OAUTH_CLIENT_SECRET: ${{ secrets.WEB_OAUTH_CLIENT_SECRET }}
+        DIM_API_KEY: ${{ secrets.DIM_API_KEY }}
```

Comment:
Need to set `CLOUDFLARE_EMAIL`, `CLOUDFLARE_KEY`, `REMOTE_HOST`, and a bunch more.

Reply:
Done